### PR TITLE
Update craftings

### PR DESCRIPTION
**Pull request by Gianluxx**
### Description
Update Craftings

### Reason to modify
^^^^

### Tests & Reviews
I have tested the code and it works.

Added some craftings, removed duplicates (wool was written 4 times->deleted 3), fixed some

### CODE REVIEW

File: src/pocketmine/inventory/CraftingManager.php [hunk 1]
```diff
@@ -419,8 +424,8 @@ public function __construct(bool $useJson = false){
 			$this->registerRecipe((new BigShapedRecipe(Item::get(Item::BOOK, 0, 1),
 				"   ",
 				"PP ",
-				"P  "
-			))->setIngredient("P", Item::get(Item::PAPER, 0, 3)));
+				"PL "
```
Comment (dktapps) [inline src/pocketmine/inventory/CraftingManager.php]: This is incorrect. The recipe you've given is the PC recipe.

Comment (Gianluxx) [inline src/pocketmine/inventory/CraftingManager.php]: is correct.. i wrote by myself and works

Comment (Gianluxx) [inline src/pocketmine/inventory/CraftingManager.php]: why can you create a book without leather? it's stupid

Comment (dktapps) [inline src/pocketmine/inventory/CraftingManager.php]: That's the PC recipe  This is a PE server.

Comment (dktapps) [inline src/pocketmine/inventory/CraftingManager.php]: Hmm, they changed it in a recent update...

**Pull request merged**